t/api/08-jobtemplates.t

```diff
@@ -1074,7 +1074,9 @@ subtest 'Create and modify groups with YAML' => sub {
                 id => $job_group_id3,
                 job_group_id => $job_group_id3,
                 error_status => 400,
-                error => ["Product 'opensuse-13.1-DVD-i586' is invalid"],
+                error => [
+"Product 'opensuse-13.1-DVD-i586' not found in database (arch: 'i586', distri: 'geeko', flavor: 'DVD', version: '13.1')"
```

Comment:
Splitting it up could possibly make it work:
```suggestion
"Product 'opensuse-13.1-DVD-i586' not found in database "
. "(arch: 'i586', distri: 'geeko', flavor: 'DVD', version: '13.1')"
```
But I wouldn't spend too much time on it. I like that I can leave the tedious work to perltidy more than some occasional disagreement on style :)